nemo-cli/src/main.rs

```diff
@@ -298,7 +336,7 @@ fn run(mut cli: CliApp) -> Result<(), CliError> {
 
     let mut stdout_used = false;
 
-    if !export_manager.write_disabled() {
+    if !export_manager.write_disabled() || cli.output.dumping.is_some() {
```

Comment:
Not dumping to stdout also activates the usual export even if this `export_manager.write_disabled()` is true. This does not seem right to me and is likely not expected by a user.
Could this not be completely separate?